lib/membrane_ogg/payloader/opus.ex

```diff
@@ -95,6 +104,12 @@ defmodule Membrane.Ogg.Payloader.Opus do
 
   @impl true
   def handle_caps(:input, caps, _ctx, state) do
+    if caps.channels > 2,
+      do:
+        IO.warn(
```

Comment:
```suggestion
        Membrane.Logger.warn(
```
If the output will be corrupted, I would also consider raising it this case